jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -719,3 +720,24 @@ function configure_server_state() {
 function generate_random_id() {
     cat /dev/urandom | env LC_CTYPE=C tr -dc 'a-zA-Z0-9' | fold -w 16 | head -n 1
 }
+
+function configure_jbpm_cluster(){
+    if [ "${KIE_SERVER_JBPM_CLUSTER^^}" = "TRUE" ]; then
+      configure_jbpm_cache
+      log_info "KIE_SERVER_JBPM_CLUSTER enabled"
+    else
+      log_info "KIE_SERVER_JBPM_CLUSTER disabled"
+    fi
+}
+
+function configure_jbpm_cache() {
+    sed -i 's#<cache-container name="server" aliases="singleton cluster" default-cache="default" module="org.wildfly.clustering.server">#<cache-container name="jbpm">\
```

Comment:
Can you tests if Kie Server are correctly cached, can we be sure that in case when there are multiple Kie Server sets running in one OCP project and they have enabled jbpmCluster, they do not affect each other, and only Kie Servers in set see it's own replicas.

Reply:
The caches enabled by Kube_ping are per project, I've tested this behaviour for the RHPAM-3181 and each cluster use the kieserver name, and cluster in different namespaces doesnt' see each others, do you mean a test like this ?

Reply:
@jakubschwan is this question answered?

Reply:
yes, marking this conversation as resovled